python/setup.py

```diff
@@ -54,6 +54,8 @@ def run(self):
     `import [agent_pb2_proto_import]`. The latter would fail because the name is
     meant to be relative but python3 interprets it as an absolute import.
     """
+    from grpc_tools import protoc  # pylint: disable=import-outside-toplevel
```

Comment:
If this is at the top of this file, we cannot resolve the dependencies without having `grpc_tools` installed.

Reply:
Done.